src/main/webapp/app/entities/faq-category.model.ts

```diff
@@ -0,0 +1,29 @@
+export class FAQCategory {
+    public color?: string;
+
+    public category?: string;
+
+    constructor(category: string | undefined, color: string | undefined) {
+        this.color = color;
+        this.category = category;
+    }
+
+    equals(otherExerciseCategory: FAQCategory): boolean {
```

Comment:
Does this class duplicate another class called `ExerciseCategory`? I feel like I've seen this code before.

If this is necessary, I think you should rename the parameter here (`otherFaqCategory`)